src/app/parks/park-list/park-list.component.ts

```diff
@@ -1,31 +1,37 @@
-import { ChangeDetectorRef, Component, OnInit } from '@angular/core';
+import { ChangeDetectorRef, Component, OnDestroy, OnInit } from '@angular/core';
+import { ParkService } from 'app/services/park.service';
 import { IColumnObject } from 'app/shared/components/table-template/table-object';
-import { Constants } from 'app/shared/utils/constants';
+import { takeWhile } from 'rxjs/operators';
 import { ParkTableRowComponent } from './park-table-row/park-table-row.component';
 
 @Component({
   selector: 'app-park-list',
   templateUrl: './park-list.component.html',
   styleUrls: ['./park-list.component.scss']
 })
-export class ParkListComponent implements OnInit {
+export class ParkListComponent implements OnInit, OnDestroy {
+  private alive = true;
   // Component
   public loading = true;
   // This will be changed to service.
```

Comment:
remove comment, looks like service is in

Reply:
e29d540